README.md

```diff
@@ -1,70 +1,75 @@
+not explicitly select `nightly` as your default toolchain then do so now
 ```
-|-- app/
-|   |-- src/
-|       |-- main.rs
-|   |-- Cargo.toml
-|   |-- Makefile
-|   |-- build.rs
-|   +-- rust-toolchain
-|-- enclave/
-|   |-- src/
-|       |-- lib.rs
-|   |-- Cargo.toml
-|   |-- Enclave.config.xml
-|   |-- Enclave.edl
-|   |-- Enclave.lds
-|   |-- Makefile
-|   |-- Xargo.toml
-|   +-- rust-toolchain
-|-- Makefile
-+-- buildenv.mk
+$ rustup default nightly
 ```
 
-## Setting up your project
+You will now have the `cargo` build tool, as well as the nigtly `rustc` compiler
+installed on your system and may now continue setting up your environment.
 
-You need to follow a few steps to use this template properly:
-- Add your ```.rs``` files to the ```src/``` folders (```lib.rs``` / your enclave source code goes in ```enclave/src```, your host/app source code goes in ```app/src```), or modify the ```.rs``` files already included with the project
-- Add your own ```Enclave.edl``` file, or modify the one joined in the project.
-- Change the ```Cargo.toml (or/and Xargo.toml if you want to use Xargo)``` files depending on of your needs (adding/removing dependencies).
-    - Be careful if you want to change the library name on the ```Cargo.toml``` file (enclave part), you will need to reflect this change on the enclave ```Makefile```, more specifically on the ```ENCLAVE_CARGO_LIB``` variable, and on the ```lib.rs``` file.
-    - If you need to change the app/host name, please make sure to edit the host ```Makefile```, and change the variable ```APP_U```.
+### Environment
 
-## Build your project
+**NOTE:** If you opted to install `gcc` as your compiler, make sure you run
+```
+export CC=gcc; export CXX=g++
+```
+Otherwise you may safely continue.
 
-### Before starting the building process, please make sure you downloaded the Rust SGX SDK repository, we're going to need the EDL and headers files joined in the SDK.
+The Rust and Intel SGX SDKs need to be installed and the
+relevant environment variables need to be set.  In order to facilitate this, we
+use the convenience scripts provided at [rust-sgx-sdk-env].
 
-Once you downloaded the Rust SGX SDK, you have multiple ways to start the building process: 
-- Run this command: ```CUSTOM_EDL_PATH=~/teaenclave/edl CUSTOM_COMMON_PATH=~/teaenclave/common make``` (replace ```~/teaenclave``` by the actual SDK location)
-- You can also run the command export (```export CUSTOM_EDL_PATH=~/teaenclave/edl```), and specify the variables before calling make. It is advised to add this command on your ```.bashrc``` file (if you use bash), or your favorite shell configuration file.
+1. Make sure the `docker` daemon is running, otherwise start it with
+   ```
+   $ systemctl start snap.docker.dockerd
+   ```
+2. In order to run the scripts as a non-root user, follow the
+   [docker-postinstall](post-installation instructions) set out in the Docker
+   documentation (note, in particular, that a restart may be necessary).
```

Comment:
Can we rather add these two points into the [rust-sgx-sdk-env] README?

Then we can replace all these 4 points with:
1. Clone the project repository
   ```
   $ git clone https://github.com/PiDelport/rust-sgx-sdk-dev-env
   ```
   and `cd` into it.
2. Follow the README in [rust-sgx-sdk-dev-env] to setup your environment.

Reply:
We can add it to both, but I will prefer if we keep all of the relevant instructions in this Readme as well.

Reply:
I cannot say for certain which approach is "better".  In this specific instance, my main concern was providing the path of least resistance, both for the DD reviewers and potential newcomers.  

To be more specific, there are other ways to set up Docker and, in fact, this is not even my preferred one.  Normally, you might rather leave it up to the developer to set it up however they want.

Reply:
Ok cool then you can just add the point `1. Clone the project repository and cd into it` at step 3 so that the reader knows where the script is from exactly.